eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1484,6 +1484,76 @@ public function testCopySubtree()
     }
 
     /**
+     * Test for the copySubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::copySubtree()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testCopySubtreeWithAliases()
+    {
+        $repository = $this->getRepository();
+        $urlAliasService = $repository->getURLAliasService();
+
+        // $mediaLocationId is the ID of the "Media" page location in
+        // an eZ Publish demo installation
+
+        // $demoDesignLocationId is the ID of the "Demo Design" page location in an eZ
+        // Publish demo installation
+        $mediaLocationId = $this->generateId('location', 43);
+        $demoDesignLocationId = $this->generateId('location', 56);
+
+        $locationService = $repository->getLocationService();
+        $locationToCopy = $locationService->loadLocation($mediaLocationId);
+        $newParentLocation = $locationService->loadLocation($demoDesignLocationId);
+
+        // Copy location "Media" to "Demo Design"
+        $copiedLocation = $locationService->copySubtree(
+            $locationToCopy,
+            $newParentLocation
+        );
+
+        // Load copied subtree items
+        $locationList = $locationService->loadLocationChildren($copiedLocation);
+
+        $expectedAliases = [
+            '49-59b514174bffe4ae402b3d63aad79fe0' => [
```

Comment:
I don't know if this is the right approach to test those data, have someone some thoughts about this?